src/ensembl/src/content/app/entity-viewer/state/gene-view/entityViewerGeneViewState.ts

```diff
@@ -0,0 +1,55 @@
+export type EntityViewerGeneFunctionState = {
+  selectedTabName: GeneFunctionTabName;
+};
+
+export type EntityViewerGeneRelationshipsState = {
+  selectedTabName: GeneRelationshipsTabName;
+};
+
+export type EntityViewerGeneViewObjectState = {
+  selectedGeneTabName: GeneViewTabName;
+  geneFunction: EntityViewerGeneFunctionState;
+  geneRelationships: EntityViewerGeneRelationshipsState;
+};
+
+export type EntityViewerGeneViewState = Readonly<{
+  [activeGenomeId: string]: {
+    [activeObjectId: string]: EntityViewerGeneViewObjectState;
```

Comment:
`activeGenomeId` → `genomeId`
`activeObjectId` → `geneId`

Reply:
That's done